src/api/initialization.ts

```diff
@@ -36,7 +36,7 @@ const contractArtifacts = [
 ].map(name => require(`../../build/contracts/${name}.json`))
 
 class DutchExchangeInit {
-  contracts: Object | any
+  contracts: object | any
```

Comment:
`object | any === any`
Not much point in it. Feel free to add something like 
```
interface Contracts{ DutchExchange: any, DutchExchangeFactory: any }
```
to types

Reply:
yeah there're probably alot of these kinds of ts typings laying around from older code ill hunt through these commit specific ones